website/docs/api/language.mdx

```diff
@@ -333,6 +333,34 @@ and custom registered functions if needed. See the
 | `component_cfg` | Optional dictionary of keyword arguments for components, keyed by component names. Defaults to `None`. ~~Optional[Dict[str, Dict[str, Any]]]~~ |
 | **RETURNS**     | The updated `losses` dictionary. ~~Dict[str, float]~~                                                                                          |
 
+## Language.distill {id="distill",tag="method",version="4"}
```

Comment:
```suggestion
## Language.distill {id="distill",tag="method,experimental",version="4"}
```
To keep it in line with [this PR](https://github.com/explosion/spaCy/pull/12016/files).